### DISCUSSION TITLE
RTP port range

### DISCUSSION THREAD

**Discussion by nicolastmaia** [sipsorcery-org/sipsorcery#840] [Q&A] [answered]
Hi.
In a webrtc connection I know we're able to specify the websocket port we want the server running sipsorcery to communicate in, but what about the RTP ports it uses if it's in a local network (no ICE involved)?
Are there default ports for webrtc? Would it be those? 

I know #439 asks something similar, but the OS Ephemeral port range (mentioned there) is pretty extensive. I wanted to know if there is a more specific range.

Thanks in advance!

**Comment by sipsorcery** [comment, marked as answer]
You can use the PortRange parameter on the RTCPeerConnection [constructor](https://github.com/sipsorcery-org/sipsorcery/blob/f4fccb11aa62bff3e0b2d38f35a947035162593e/src/net/WebRTC/RTCPeerConnection.cs#L386) if you want a custom port range.

As for default RTP ports for WebRTC, there aren't any.

**Reply by nicolastmaia** [reply]
Great! Thanks!
